spotbugs/src/main/java/edu/umd/cs/findbugs/detect/MutableStaticFields.java

```diff
@@ -82,6 +82,7 @@ static String extractPackage(String c) {
 
     static boolean mutableSignature(String sig) {
```

Comment:
This method needs to be rewritten to cover much more classes, but in a separate pull request.